content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,16 @@
+## Bug fixes and minor enhancements
+
+- Fix duplicate CRD being generated in the install ([Issue 14976](https://github.com/istio/istio/issues/14976))
+- Allow users to disable istio default retries by setting retries to 0 ([Issue 14900](https://github.com/istio/istio/issues/14900))
+- Improve citadel log ([Pull Request 15032](https://github.com/istio/istio/pull/15032))
+- Add citadel replicaCount flag back and workloadCertTtl to support multi-citadel and short cert TTL for perf test ([Pull Request 15066](https://github.com/istio/istio/pull/15066))
+- Fix Mixer CRD store issue ([Issue 14841](https://github.com/istio/istio/issues/14841))
+- Add HTTP/1.0 support to gateway configuration generation ([Pull Request 15062](https://github.com/istio/istio/pull/15062))
+- Redis filter (this feature is guarded with the feature flag PILOT_ENABLE_REDIS_FILTER, disabled by default):
+    - Enable redis filter ([Pull Request 15049](https://github.com/istio/istio/pull/15049))
+    - Default Redis Cluster ([Pull Request 15052](https://github.com/istio/istio/pull/15052))
+    - Configure Redis latency in microseconds ([Pull Request 15119](https://github.com/istio/istio/pull/15119))
+- Do not overwrite Citadel storage namespace with env var ([Pull Request 15082](https://github.com/istio/istio/pull/15082))
+- Add toleration for Istio components ([Pull Request 15081](https://github.com/istio/istio/pull/15081))
+- set nodeagent updateStrategy to RollingUpdate ([Pull Request 15129](https://github.com/istio/istio/pull/15129)) ([Issue 15020](https://github.com/istio/istio/issues/15020))
```

Comment:
- Fix cause of 'TLS error: Secret is not supplied by SDS' errors during upgrade ([Issue 15020](https://github.com/istio/istio/issues/15020)).
- Fix crash in Istio's JWT Envoy filter caused by malformed JWT ([Issue 15084](https://github.com/istio/istio/issues/15084)).

Reply:
@duderino  You are commenting the 1.2.1 release note, I was going to open another PR for 1.2.2 which included these ones. I can continue with this and rename everything to 1.2.2